docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+```--processes``` we requested using the Calibrators, Selector, and Producers
+(often referred to as CSPs) as requested. In the following examples, we will skip the
+```--calibrators```, ```selector``` and ```producers``` parameters, which means that the defaults
+defined in the config will be used automatically. Examples on how to
+implement your own CSPs can be found in the [calibrators](building_blocks/calibrators),
+[selectors](building_blocks/selectors), and [producers](building_blocks/producers) sections of the
+user guide. The ```--variables``` parameter defines, for which variables we want to create histograms
+and plots. Variables are order objects that need to be defined in the config as shown in the
+[config objects](building_blocks/config_objects) section and the column corresponding to the expression
+statement needs to be stored either after the {py:class}`~columnflow.tasks.reduction.ReduceEvents`
+For each of the category given with the ```--categories``` parameter, one plot will be produced.
+A detailed guide on how to implement categories in Columnflow is given in the
+[categories](building_blocks/categories) section.
+
+To define, which processes and datasets to consider when plotting, you can use the ```--processes```
+and ```--datasets``` parameter. When only processes are given, all datasets corresponding to the
+requested processes will be considered. When only datasets are given, all processes in the config
+will be considered.
+The ```processes``` parameter can be used to change the order of processes in the stack and the legend
+(try for example ```--processes st,tt``` instead) and to further distinguish between sub-processes
+(e.g. via ```--processses, tt_sl,tt_dl,tt_fh```).
+
+:::{dropdown} Note! Do not add the same dataset via multiple processes!
+At the time of writing this documentation, there is still an issue present that histograms corresponding
+to a dataset can accidentally be used multiple times. For example, when adding ```--processes tt,tt_sl```,
+the events corresponding to the dataset ```tt_sl_powheg``` will be displayed twice in the resulting
+plot.
+:::
+
+
+## Customization of plots
+
+There are many different parameters implemented that allow customizing the style of a plot. A short
+description to all plotting parameters is given in the [Plotting tasks](../task_overview/plotting.md).
+In the following, a few exemplary task calls are given to present the usage of our plotting parameters.
+
+Per default, the PlotVariables1D task creates one plot per variable with all Monte Carlo backgrounds being included
+in a stack and data being shown as separate points. The bottom subplot shows the ratio between signal
+and all processes included in the stack and can be disabled via the ```--skip_ratio``` parameter.
+To change the text next to the label, you can add the ```--cms-label``` parameter.
+To compare shapes of multiple processes, you might want to plot each process separately as one line.
```

Comment:
It might be good to use separate paragraphs for different style options

Reply:
I feel like having a paragraph for each parameter might be overkill since some parameters are just mentioned in a single sentence. Maybe one new paragraph before discussing the `process-settings` parameter?